crates/ethereum/primitives/src/transaction.rs

```diff
@@ -280,19 +281,22 @@ impl From<TypedTransaction> for Transaction {
 }
 
 /// Signed Ethereum transaction.
-#[derive(Debug, Clone, Eq, Serialize, Deserialize, derive_more::AsRef, derive_more::Deref)]
+#[derive(
+    Debug, Clone, Eq, Serialize, Deserialize, derive_more::AsRef, derive_more::Deref, DerefMut,
```

Comment:
this should not always derefmut, 

please feature gate this impl via test-utils feature

Reply:
agreed
hint, done by
```rust
#[cfg_attr(feature = "test-utils", derive(derive_more::DerefMut))]
```